package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@kontist/mock-solaris",
-  "version": "1.0.39",
+  "version": "1.0.47",
```

Why not changing the version to the next one?
```suggestion
  "version": "1.0.40",
```

---

[Because I've made intermediary releases to be able to experiment](https://github.com/kontist/mock-solaris/tags)